assets/models/system/automl_instance_segmentation/spec.yaml

```diff
@@ -5,7 +5,7 @@ properties:
   datasets: COCO
   finetune-min-sku-spec: 4|1|28|176
   finetune-recommended-sku: Standard_NC6s_v3
-  finetuning-tasks: image-object-detection
+  finetuning-tasks: image-instance-segmentation
```

Comment:
```suggestion
  finetuning-tasks: image-segmentation
```

Reply:
filter on ui is "image-segmentation" .. :)

Reply:
discussed offline. no need to change.